.github/workflows/tests.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         typo3-version: ["10.4", "11.5"]
-        php-version: ["7.4", "8.0", "8.1", "8.2"]
```

Comment:
PHP 8.3 in combination with TYPO3 10.4 needs to be added to the exclude list (below) since it's not compatible to each other.